Mesh_3/include/CGAL/Mesh_3/tet_soup_to_c3t3.h

```diff
@@ -141,7 +151,15 @@ bool build_finite_cells(Tr& tr,
 
     Cell_handle c = tr.tds().create_cell(vs[0], vs[1], vs[2], vs[3]);
     c->set_subdomain_index(tet[4]); // the cell's info keeps the reference of the tetrahedron
-
+    c->info() = tet[4]; // the cell's info keeps the reference of the tetrahedron
```

Comment:
tetrahedral remeshing uses this function, and we do not have `info()` in cells.
Why isn't `subdomain_index()` enough?

Reply:
It seems it is after all, must have been a reminiscence from an older try